community-contributions/joystreamtelegrambot/config.ts

```diff
@@ -0,0 +1,11 @@
+// website
+export const domain = "https://testnet.joystream.org";
+
+export const wsLocation = "ws://rome-rpc-endpoint.joystream.org:9944/";
+// websocket location
```

Comment:
I don't know if this is something just on my side here but running the bot I'd get this: 

```
2020-10-30 12:14:38          API-WS: disconnected from ws://rome-rpc-endpoint.joystream.org:9944/ code: '1006' reason: 'connection failed'
```

Changing `ws` to `wss` seems to fix it.

This is nitpicking but maybe also move the comment above the declaration? :)